doc/nrf/migration/migration_guide_1.x_to_2.x.rst

```diff
@@ -0,0 +1,53 @@
+.. _ncs_2.0.0_migration:
+
+Migration notes for |NCS| v2.0.0
+################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+In version 2.0.0, the |NCS| introduced a number of breaking changes into its codebase.
+If your application was built using |NCS| v1.x.x, make sure to complete the actions listed in the following sections to migrate your application to |NCS| v2.x.x.
+
+Toolchain change
```

Comment:
@carlescufi  please provide more information

Reply:
Do we currently have any toolchain-related information in NCS @b-gent?

Reply:
Only the 'Install a Toolchain' section that we are now including from Zephyr (we are pointing to it already).
Actually, I will change the link here to point to our documentation instead of Zephyr in this matter (same content).